docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4678,7 +4682,6 @@ semantics:
   service for each clone packet instance will be set to the value programmed in
   the clone session entry (`class_of_service` field). This value must be a valid
   value for the PSA `ClassOfService_t` type, which supports runtime translation
-  by default [@PSATranslation], or the server must return
```

Comment:
that looks like an unrelated (and incorrect) change?

Reply:
Fixed up the incorrect change.